errai-demos/errai-bus-demo-stock/src/main/resources/log4j.properties

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-log4j.rootLogger=ERROR, mainlogger
+log4j.rootLogger=INFO, mainlogger
```

Comment:
I noticed that each demo module was producing a different output while building, so I thought that could be confusing and made them all use the same level, which is INFO. Most of the demos were already using INFO, so I just changed the ones that weren't. It didn't increase much the output amount and it made the build output more consistent, I think. 

It's not something I think we really need, but something that standardizes the output of all demos. If you see any negative impacts this can have, I'm ok with reverting it.

Reply:
Okay. Let's leave it at INFO for now.